cmd/aries-js-worker/vue-aries-framework-go/README.md

```diff
@@ -12,13 +12,19 @@ npm install
 npm run serve
 ```
 
+**Note:** this target requires `npm link` to run without `sudo`.
+Install `npm` via [nvm](https://github.com/nvm-sh/nvm) (recommended) or give it the proper permissions in the
+required paths.
```

Comment:
what is the required path here ? example would be helpful to understand.

Reply:
The ones that it complains about.

Sorry - don't have the specifics on hand. I no longer have its `.deb. package installed. Only via `nvm`.